api/test/wfl/system/v1_endpoint_test.clj

```diff
@@ -45,6 +45,7 @@
                 (is uuid "workloads should be been assigned a uuid")
                 (is (util/absent? uuids uuid))
                 (is (:created workload) "should have a created timestamp")
+                (is (:workflow_options workload) "should have non-null options")
```

Comment:
Options can technically be null, but for the modules we test here that will never be the case because there are always defaults that will be stored in this key